os/kernel/binary_manager/binary_manager.h

```diff
@@ -82,6 +82,8 @@ struct binmgr_bininfo_s {
 	uint16_t bin_offset;
 	uint16_t inuse_idx;
 	uint32_t part_size;
+	uint32_t bin_stack_size;
+	uint8_t bin_priority;
```

Comment:
size_t priority is 32bit while bin_priority is 8bit, consider making both of same type